dev/test/typescript.ts

```diff
@@ -33,6 +33,7 @@ import Precondition = FirebaseFirestore.Precondition;
 import SetOptions = FirebaseFirestore.SetOptions;
 import Timestamp = FirebaseFirestore.Timestamp;
 import Settings = FirebaseFirestore.Settings;
+import {defaultConverter} from '../src/types';
```

Comment:
Would it be possible to create your own converter, which will also test the types on `FirestoreDataConverter`?